build-package.ps1

```diff
@@ -9,5 +9,11 @@ Import-Module "$PSScriptRoot/scripts/ps-modules/Build" -Force -DisableNameChecki
 Write-Banner -Level 1 -Title "Installing preconfigured package: `"$PackageName`""
 $pkg = Get-PackageInfo -PackageName $PackageName
 
+if ($pkg -eq $null) {
+    $selectedSection = if ((Get-IsOnWindowsOS)) { "win" } elseif ((Get-IsOnMacOS)) { "mac" } else { "linux" }
+    Write-Message "Package $PackageName contains no section for $selectedSection, skipping build."
+    exit 0
```

Comment:
I kind of wish there were a way to not even spin up the Linux box (or whatever OS(es)) if there was not a config for it, but I'm not sure how to dynamically build a strategy matrix in the YAML.  This is probably more than sufficient for now, though.  We don't intend to run these builds that often, other than maybe just testing out a new package or feature or something.

Reply:
I was thinking the same, but nothing came to mind.  At least it only runs for around 13 seconds.